filesystem/iso9660.ksy

```diff
@@ -5,269 +5,607 @@ meta:
+        if: _parent.expiration
+      - id: effective
+        type: datetime_short
+        if: _parent.effective
+  rrip_tf_long:
+    doc-ref: rrip 4.6.1
+    seq:
+      - id: creation
+        type: datetime_long
+        if: _parent.creation
+      - id: modify
+        type: datetime_long
+        if: _parent.modify
+      - id: access_short
+        type: datetime_long
+        if: _parent.access
+      - id: attributes
+        type: datetime_long
+        if: _parent.attributes
+      - id: backup
+        type: datetime_long
+        if: _parent.backup
+      - id: expiration
+        type: datetime_long
+        if: _parent.expiration
+      - id: effective
+        type: datetime_long
+        if: _parent.effective
+  rrip_tf:
+    doc-ref: rrip 4.6.1
+    seq:
+      - id: length
         type: u1
-      - id: month
+      - id: version
+        contents: [ 0x1 ]
+      - id: long_form
+        type: b1
+      - id: effective
+        type: b1
+      - id: expiration
+        type: b1
+      - id: backup
+        type: b1
+      - id: attributes
+        type: b1
+      - id: access
+        type: b1
+      - id: modify
+        type: b1
+      - id: creation
+        type: b1
+      - id: datetime_short
+        type: rrip_tf_short
+        if: not long_form
+      - id: datetime_long
+        type: rrip_tf_long
+        if: long_form
+  susp_unknown: # default for now
+    seq:
+      - id: length
         type: u1
-      - id: day
+      - id: version
         type: u1
-      - id: hour
+      - id: data
+        size: length - 4
+  su_header:
+    seq:
+      - id: signature
+        type: u2be
+        enum: su_signature
+      - id: su_specific
+        type:
+          switch-on: signature
+          cases:
+            'su_signature::rras_amiga_specific': susp_unknown # AS
+            'su_signature::susp_continuation_area': susp_unknown # CE
+            'su_signature::rrip_child_link': susp_unknown # CL
+            'su_signature::susp_extensions_reference': susp_unknown # ER
+            'su_signature::susp_extension_selector': susp_unknown # ES
+            'su_signature::rrip_alternate_name': rrip_nm # NM
+            'su_signature::susp_padding_field': susp_unknown # PD
+            'su_signature::rrip_parent_link': susp_unknown # PL
+            'su_signature::rrip_posix_device_number': susp_unknown # PN
+            'su_signature::rrip_posix_file_attributes': rrip_px # PX
+            'su_signature::rrip_relocated_directory': susp_unknown # RE
+            'su_signature::rrip_extensions_in_use_indicator': susp_unknown # RR
+            'su_signature::rrip_sparse_file': susp_unknown # SF
+            'su_signature::rrip_symbolic_link': susp_unknown # SL
+            'su_signature::susp_indicator': susp_unknown # SP
+            'su_signature::susp_terminator': susp_unknown # ST
+            'su_signature::rrip_time_file': rrip_tf # TF
+            'su_signature::rrzf_zisofs': susp_unknown # ZF
```

Comment:
If `susp_unknown` is supposed to catch all "yet unsupported" types, then why don't we just use `_` (i.e. else/default case)?

```yaml
cases:
  'su_signature::rrip_alternate_name': rrip_nm # NM
  'su_signature::rrip_posix_file_attributes': rrip_px # PX
  'su_signature::rrip_time_file': rrip_tf # TF
  _: susp_unknown
```

Reply:
I guess they should be grouped (as a draft for future improvements) and commented out.

Reply:
That would look nice in code but... it breaks the SU stream on ISO's without SUSP

Reply:
Understood, thanks. I'll try to think what could be done here...

Reply:
Maybe a defaults for:
in the enum, but not specific overruled
not found in the enum